core/modules/wdb/python/schema_migration.py

```diff
@@ -0,0 +1,211 @@
+"""Module defining methods used in schema migration of qservw_worker database.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__all__ = ["make_migration_manager"]
+
+#--------------------------------
+#  Imports of standard modules --
+#--------------------------------
+import logging
+import os
+import re
+
+#-----------------------------
+# Imports for other modules --
+#-----------------------------
+from lsst.qserv.schema import SchemaMigMgr
+from lsst.db import utils
+
+#----------------------------------
+# Local non-exported definitions --
+#----------------------------------
+
+_log = logging.getLogger(__name__)
+
+#------------------------
+# Exported definitions --
+#------------------------
+
+
+class WdbMigrationManager(SchemaMigMgr):
```

Comment:
If this is a copy a copy of migration class from `qmeta` then we should move it into `schema` module and reuse in both modules.